### DISCUSSION TITLE
How to define and fine-tune our own task?

### DISCUSSION THREAD

**Discussion by GeorgeOrfanidis** [apache/incubator-mxnet#19552] [Q&A] [answered]
Hello everybody, 

I am new at using GLUON for sentence pair classification and I would like to ask for your help. I initially found this very simple tutorial for sentence pair classification - https://nlp.gluon.ai/examples/sentence_embedding/bert.html. However, for simplicity, this tutorial omits important stages and steps such as warmup learning rate schedule and validation on the dev dataset. Would it be possible to explain how to use our own dataset to finetune the Bert model using GLUON? I was going over the finetune_classifier.py file and includes MRPC, QQP, QNLI, RTE, STS-B, CoLA, MNLI, WNLI, SST tasks name to fine-tune, how can I create and fine-tune my own task using the dataset that I have created?

Thank you in advance!

**Comment by leezu** [comment, marked as answer]
You can refer to the scripts at https://github.com/dmlc/gluon-nlp/tree/v0.10.x/scripts for MXNet 1.x and https://github.com/dmlc/gluon-nlp/tree/master/scripts for the MXNet 2 development version (master branch).

**Reply by GeorgeOrfanidis** [reply]
Hey and thank you very much for your response. Would it be possible to further guide me? Can you point me to some specific file if possible? 

Thank you in advance

**Reply by leezu** [reply]
Sure. For MXNet 1 you can take a look at https://github.com/dmlc/gluon-nlp/blob/v0.10.x/scripts/bert/finetune_classifier.py
If you have more detailed questions I recommend you open a new discussion at https://github.com/dmlc/gluon-nlp
Thank you

**Reply by GeorgeOrfanidis** [reply]
Thank you for your help. I have created a new discussion thread!
https://github.com/dmlc/gluon-nlp/discussions/1438
